src/lib/djerba/plugins/plugin_tester.py

```diff
@@ -78,8 +78,10 @@ def run_basic_test(self, test_source_dir, params,
         self.assertTrue(validator.validate_data(plugin_data_found))
         self.assertEqual(plugin_data_found, plugin_data_expected)
         # TODO check other document types, eg. research
+        # this will break if there is more than one attribute per json
```

Comment:
Can we check and give an informative error if there are multiple attributes? Eg.
`if len(plugin_data_found['attributes']) != 1:
    raise RuntimeError('Test framework requires exactly 1 attribute, found: {0}'.format(plugin_data_found['attributes'])))
`